src/videojs-contrib-hls.js

```diff
@@ -342,7 +342,9 @@ class HlsHandler extends Component {
     // start playlist selection at a reasonable bandwidth for
     // broadband internet
     // 0.5 MB/s
-    this.options_.bandwidth = this.options_.bandwidth || 4194304;
+    if (typeof this.options_.bandwidth === 'undefined') {
```

Comment:
People can put all sorts of terrible things in the configuration. I think checking if `bandwidth` is a number is a safer approach here.